docs/AboutUs.md

```diff
@@ -4,49 +4,72 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ## Project Team
 
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
+#### [Chuan Wei Candiie](https://github.com/Candiie) <br>
+<img src="images/Candiie.png", width="150"><br>
 **Role**: Project Advisor
 
------
-
-#### [Joshua Lee](http://github.com/lejolly)
-<img src="images/JoshuaLee.jpg" width="150"><br>
-Role: Developer <br>  
-Responsibilities: UI
+----- 
 
------
+#### [Lee Wenwei Johnervan](http://github.com/johnervan) <br>
+<img src="images/JohnervanLee.png", width="150"><br>
+**Role**: Team Lead <br>
+Responsible for:
+* Coordinating documentation.
+* Managing and leading the project team.
+* Ensuring that the project is delivered on time.
+* Maintaining a co-operative, motivated and successful team.
+* Continuously learn and improve skills.
 
-#### [Leow Yijin](http://github.com/yijinl) 
-<img src="images/LeowYijin.jpg" width="150"><br>
-Role: Developer <br>  
-Responsibilities: Data
+Tool Expert:
+* Windows Environment Tester
+* JUnit
+* EclEmma
 
 -----
 
-#### [Martin Choo](http://github.com/m133225)
-<img src="images/MartinChoo.jpg" width="150"><br>
-Role: Developer <br>  
-Responsibilities: Dev Ops
+#### [Calvin Yang Jiawei](http://github.com/origiri) <br>
+<img src="images/CalvinYang.png", width="150"><br>
+**Role**: Developer <br>
+Responsible for:
+* Coordinating storage.
+* Contributing to overall project objectives and specific team deliverables.
+* Ensuring the proper integration of the project management processes.
+* Continuously learn and improve skills.
+
+Tool Expert:
+* SourceTree
+* Eclipse
 
 -----
 
-#### [Thien Nguyen](https://github.com/ndt93)
- Role: Developer <br>  
- Responsibilities: Threading
- 
- -----
+#### [Chia Wei Kang](http://github.com/weikangchia) <br>
+<img src="images/ChiaWeiKang.png", width="150"><br>
+**Role**: Developer <br>
+Responsible for:
+* Contributing to overall project objectives and specific team deliverables.
+* Coordinating testing.
+* Ensuring that the project is delivered to the quality standard.
+* Analysing the system-in-use so as to continually refine usability and the user interface.
+* Continuously learn and improve skills.
 
-#### [You Liang](http://github.com/yl-coder) 
-<img src="images/YouLiang.jpg" width="150"><br>
- Role: Developer <br>  
- Responsibilities: UI
- 
- -----
+Tool Expert:
+* Regex
+* Travis
+* Slack
+
+-----
 
-# Contributors
+#### [Foo En Teng Joel](http://github.com/jaeoheeail) <br>
+<img src="images/JoelFoo.png", width="150"><br>
+**Role**: Developer <br>
+Resonsible for:
+* Contributing to overall project objectives and specific team deliverables.
```

Comment:
@johnervan @jaeoheeail @weikangchia @origiri great job updating and specifying the scopes as suggested ~ Lets all work hard and - Continuously learn and improve skills - for the rest of this semester XD